Game/Src/UI/HealthUI.gd

```diff
@@ -1,20 +1,25 @@
 extends Control
 
 onready var stats = $StatsGen
-
 onready var green =$Green
 onready var red = $Red
+onready var size = red.rect_size.x
+export var max_health = 100
+
+export(PackedScene) var statsScript
 
-var max_damage
+var health_unit
```

Comment:
It is not clear what health_unit contains.
You compute size (I would rename it to red_bar_size) and at start up, you compute the fraction of health already taken as damage? If so, shouldn't that be zero since the red bar is originally of size 0?

Reply:
Nope sorry for my lack of comment. Health_unit, (as glory_unit for glory_UI) gives us the pixel size of one unit of health. Thus, it is size/max_health

Reply:
I would rename health_unit into px_per_hp then, what do you think?